README.md

```diff
@@ -17,9 +17,40 @@ A few key features:
 
 ## Getting Up and Running
 
-[Using a docker container](docs/docker_container.md)
+Start using OpenVino&trade; Model Server in 5 Minutes or less
 
-[Landing on bare metal or virtual machine](docs/host.md)
+```bash
+# Download the OpenVino Model Server image
```

Comment:
Change to `# Download the latest Model Server image`

Reply:
done